src/localPRG.cpp

```diff
@@ -1319,22 +1323,23 @@ float gaps(std::vector<uint32_t> v1, std::vector<uint32_t> v2, const uint32_t &m
 }
 
 
-void LocalPRG::add_sample_covgs_to_vcf(VCF &vcf, const KmerGraph &kg, const std::vector<LocalNodePtr> &ref_path,
+void LocalPRG::add_sample_covgs_to_vcf(VCF &vcf, const KmerGraphWithCoverage &kg, const std::vector<LocalNodePtr> &ref_path,
                                        const uint32_t &min_kmer_covg, const std::string &sample_name,
                                        const uint32_t &sample_id) const {
-    BOOST_LOG_TRIVIAL(info) << "Update VCF with sample covgs";
+    BOOST_LOG_TRIVIAL(debug) << "Update VCF with sample covgs";
 
     assert(!prg.nodes.empty()); //otherwise empty nodes -> segfault
     vcf.sort_records();
 
     std::vector<LocalNodePtr> alt_path;
 
     std::vector<KmerNodePtr> ref_kmer_path = kmernode_path_from_localnode_path(ref_path);
-    std::cout << "ref path: ";
+
```

Comment:
If these outputs aren't required then just delete them or add them to logging at a very low level such as `trace`

Reply:
I am not sure if they are required somehow, I will leave this to @rmcolq , as she probably knows better.

Reply:
removed in commit 7e05574650f90892364d810874c9769ab14eb3b1